catroidTest/src/org/catrobat/catroid/common/bluetooth/BluetoothConnectionProxy.java

```diff
@@ -34,14 +34,14 @@
 
 class BluetoothConnectionProxy implements BluetoothConnection {
 
-	Logger logger;
+	static BluetoothLogger logger;
```

Comment:
This should be final, not static (even if pmd may say so). It's initialized with the constructor.